src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(x) {
+      this.result += x;
+    },
+    subtract(x) {
+      this.result -= x;
+    },
+    multiply(x) {
+      this.result *= x;
+    },
```

Comment:
```suggestion
    add(x) {
      this.result += x;
    },
    
    subtract(x) {
      this.result -= x;
    },
    
    multiply(x) {
      this.result *= x;
    },
```
Would be great to add newlines between methods for better readability)